monitoring/osd-managing-metrics.adoc

```diff
@@ -34,5 +39,7 @@ include::modules/osd-monitoring-exploring-the-visualized-metrics.adoc[leveloffse
 * xref:../monitoring/osd-troubleshooting-monitoring-issues.adoc#osd-troubleshooting-monitoring-issues[Troubleshooting monitoring issues]
 
 [id="managing-metrics-next-steps"]
-.Next steps
-xref:../monitoring/osd-managing-alerts.adoc#osd-managing-alerts[Alerts]
+== Next steps
+* xref:../monitoring/osd-managing-alerts.adoc#osd-managing-alerts[Alerts]
+
+// TODO: Why is alerts a next step if alerts aren't supported? Can this be removed?
```

Comment:
I'll check with JQ on this. She wrote this topic and I wasn't part of the review.